subjects/loafofbread/README.md

```diff
@@ -16,29 +16,25 @@ func LoafOfBread(str string) string {
 
 ### Usage
 
-Here is a possible program to test your function :
+Here is a possible program to test your function:
 
 ```go
 package main
 
 import "fmt"
 
 func main() {
-	LoafOfBread("deliciousbread")
-  LoafOfBread("This is a loaf of bread")
-  LoafOfBread("Bread crumbles")
   fmt.Print(LoafOfBread("deliciousbread"))
   fmt.Print(LoafOfBread("This is a loaf of bread"))
   fmt.Print(LoafOfBread("loaf"))
 }
 ```
 
-And its output :
+And its output:
 
 ```go
 $ go run . | cat -e
 delic ousbr ad$
-Thisi ashor sente ce$
-This  s a l af of bread$
+Thisi aloaf fbrea$
```

Comment:
According to the description, spaces should be ignored when in the middle of the word, in this case the space isn't in the middle of the word, shouldn't the space count? In that case the `o` of `of` should be there, right?

Either that, or explicitly saying that all spaces should be ignored, but it would be more fun the original way.

Reply:
The required changes are fixed.

Reply:
Thanks